### PR TITLE
Fix C arrow markup

### PR DESCRIPTION
**Pull request by krOoze**
Some overlap with #1215 (sorry).

### CODE REVIEW

File: chapters/clears.txt [hunk 1]
```diff
@@ -155,20 +155,20 @@ ifndef::VK_VERSION_1_2,VK_EXT_separate_stencil_usage[]
 endif::VK_VERSION_1_2,VK_EXT_separate_stencil_usage[]
 ifdef::VK_VERSION_1_2,VK_EXT_separate_stencil_usage[]
   * [[VUID-vkCmdClearDepthStencilImage-pRanges-02658]]
-    If any element of pname:pRanges.aspect includes
+    If pname:aspect member of any element of pname:pRanges includes
```
Comment (oddhack) [inline chapters/clears.txt:158]: "If the aspect member..." here and below.

Comment (krOoze) [reply chapters/clears.txt]: done

**Comment by oddhack** [comment]
We have a large set of internal MRs that will be published in a few days in the next spec update, so I suggest holding off on resolving conflicts in this PR until that update is out.

**Comment by oddhack** [comment]
OK, 1.2.137 spec update is out - if you'll resolve conflicts I can take this now. Thanks!

**Comment by krOoze** [comment]
@oddhack rebased.

File: chapters/commonvalidity/build_acceleration_structure_common.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // Common to acceleration structure build commands
 
   * [[VUID-{refpage}-pOffsetInfos-03402]]
-    Each element pname:ppOffsetInfos[i] must: be a valid pointer to an array of
+    Each element of pname:ppOffsetInfos[i] must: be a valid pointer to an array of
```
Comment (krOoze) [inline chapters/commonvalidity/build_acceleration_structure_common.txt:5]: @oddhack actually this was somewhat intentional. It is a single hypotetical element `array[i]`, not an array.

**Pull request merged**